internal/admin/handle_authorized_apps.go

```diff
@@ -54,11 +54,14 @@ func (s *Server) HandleAuthorizedAppsSave() func(c *gin.Context) {
 					ErrorPage(c, "Invalid request, app to edit not found.")
 					return
 				}
+				if authApp == nil {
```

Comment:
Twas bug. This package returns nil if the app doesn't exist. I'm not sure if we should audit for instances or change the behavior of GetAuthorizedApp.